app/src/main/java/com/myvetpath/myvetpath/Group.java

```diff
@@ -0,0 +1,60 @@
+package com.myvetpath.myvetpath;
+
+import java.sql.Date;
+
+public class Group {
+    //Table Fields
+    public static final String TABLE_NAME = "Group";
```

Comment:
We can't have Group as a table name because it is a reserved word in SQL. If we just change it to **Groups** it's fine.